src/main/webapp/blur.js

```diff
@@ -72,3 +73,42 @@ function getAverageColor(image) {
 
   return rgbToHex(...averageColorRgb);
 }
+
+/**
+ * Function to get image with some areas blurred.
+ * @param {Array<Rect>}rectsToBlur Areas to blur.
+ * @param {Image} image
+ * @param {Number} blurRadius
+ * @return {HTMLCanvasElement}
+ */
+function getImageWithBlurredAreas(rectsToBlur, image, blurRadius) {
```

Comment:
I think it would be better if you added a little explanation of the implementation of this function, as it's not very easy to understand in my opinion.